### DISCUSSION TITLE
Is it possible to add icon/checkbox to Menu item?

### DISCUSSION THREAD

**Discussion by xahon** [tauri-apps/tauri#9338] [Q&A] [answered]
Ideally I want to add both like in kdiff3 app
![image](https://github.com/tauri-apps/tauri/assets/18731681/469d8f55-a87a-4f30-90ef-79d10927d07e)

**Comment by FabianLars** [comment, marked as answer]
icon menu items will be supported in v2. "normal" checkbox menu items are supported in v1 and v2. But a mix of both is not supported as far as i know. Feel free to open a feature request (as a normal github issue) for this :)

**Reply by xahon** [reply]
Is it supported in v2.0.0-beta.13?
I think I can work around with using enabled and disabled versions of icon

**Reply by FabianLars** [reply]
Yes
